### PR TITLE
Updated README.md to add a Travis build status image, fixed Travis build

### PR DESCRIPTION
**Pull request by javabrett**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
+dist: trusty
+addons:
+  apt:
+    packages:
+    - fakeroot
 language: scala
 script:
-  - sbt -Dsbt.build.version=1.0.0-M1 ++$TRAVIS_SCALA_VERSION rpm:packageBin debian:packageBin universal:packageBin
+  - sbt -Dsbt.build.version=0.13.16 ++$TRAVIS_SCALA_VERSION rpm:packageBin debian:packageBin universal:packageBin
```
Comment (dwijnand) [inline .travis.yml]: I'm unsure (unclear) if this is right. @eed3si9n do you remember what this is for?

Comment (eed3si9n) [reply .travis.yml]: Not sure. The change was introduced here - https://github.com/sbt/sbt-launcher-package/pull/110

Comment (javabrett) [reply .travis.yml]: Right you are, done, thanks.

File: project/build.properties [hunk 2]
```diff
@@ -1,2 +1,2 @@
-sbt.version=0.13.15
+sbt.version=0.13.16
```
Comment (javabrett) [inline project/build.properties]: Note that this is as far as the actual build SBT can go for now - can't move to 1.0.3 as there are missing dependencies for 1.x:

```
[error] sbt.librarymanagement.ResolveException: unresolved dependency: com.typesafe.sbt#sbt-native-packager;1.2.0-M9: not found
[error] unresolved dependency: net.databinder#dispatch-http_2.12;0.8.10: not found
[error] unresolved dependency: com.eed3si9n#sbt-export-repo;0.1.0: not found
[error] unresolved dependency: com.jsuereth#sbt-pgp;1.0.1: not found
```

Comment (dwijnand) [reply project/build.properties]: That's fine, we can continue to use (for now) sbt 0.13 to build sbt 1 releases.

**Pull request merged**